adafruit_vl53l0x.py

```diff
@@ -45,6 +45,7 @@
   https://github.com/adafruit/circuitpython/releases
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
+import atexit
```

Comment:
atexit isn't always available, please try to make the changes you want without atexit in the library

Reply:
Ouuu I see, hmm perhaps the continuous mode should be stopped manually from program instead from the library?

Reply:
yep!